server/services/licenceService.ts

```diff
@@ -22,39 +24,54 @@ const {
   getFieldName,
 } = require('../utils/functionalHelpers')
 
-module.exports = function createLicenceService(licenceClient) {
-  async function reset() {
+export interface LicenceRecord {
+  licence: Licence
+  stage: string
+  version: string
+  versionDetails: { version: number; vary_version: number }
+  approvedVersion: string
+  approvedVersionDetails: { version: number; vary_version: number }
+}
+
+export class LicenceService {
+  private readonly licenceClient: LicenceClient
+
+  constructor(licenceClient: LicenceClient) {
+    this.licenceClient = licenceClient
+  }
+
+  async reset() {
     try {
-      await licenceClient.deleteAll()
+      await this.licenceClient.deleteAll()
     } catch (error) {
       logger.error('Error during reset licences', error.stack)
       throw error
     }
   }
 
-  async function getLicence(bookingId) {
+  async getLicence(bookingId: number): Promise<LicenceRecord> {
     try {
-      const details = await Promise.all([
-        licenceClient.getLicence(bookingId),
-        licenceClient.getApprovedLicenceVersion(bookingId),
+      const [rawLicence, rawVersionDetails] = await Promise.all<CaseWithVaryVersion, ApprovedLicenceVersion>([
+        this.licenceClient.getLicence(bookingId),
+        this.licenceClient.getApprovedLicenceVersion(bookingId),
       ])
 
-      const rawLicence = details[0]
-      const rawVersionDetails = details[1]
+      if (!rawLicence) {
+        return null
+      }
 
-      const licence = getIn(rawLicence, ['licence'])
+      const { licence, stage } = rawLicence
       if (!licence) {
         return null
       }
-      const formattedLicence = formatObjectForView(licence)
-      const approvedVersionDetails = rawVersionDetails ? formatObjectForView(rawVersionDetails) : {}
```

Comment:
I can't see how {formatObjectForView} could change {licence} or {rawVersionDetails} so I've removed both invocations.  All the tests are passing, I've tried checking the code for the field names used within that function and haven't found any matches.  

Is this risky???